### PR TITLE
Update Ember CLI

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR **updates** the **Ember CLI** version using the automated command [`ember-cli-update`](https://github.com/ember-cli/ember-cli-update).

Command used: `ember-cli-update --from 2.18.0`

---

~In order to pass eslint, add as globals the globally-defined test methods in acceptance tests.~ Updated the acceptance test to latest ember.

### CODE REVIEW

**Comment by olleolleolle** [comment]
# Test failures

```
not ok 2 Chrome 67.0 - TestLoader Failures: dummy/tests/acceptance/string-config-test: could not be loaded
    ---
        actual: >
            null
        stack: >
            Error: Could not find module `dummy/tests/helpers/module-for-acceptance` imported from `dummy/tests/acceptance/string-config-test`
```

**Comment by olleolleolle** [comment]
I looked at the guide, and began shaping the thing to be like that.
https://guides.emberjs.com/release/testing/acceptance/ 

There's still 1 failing test.

**Comment by olleolleolle** [comment]
@mriska Acceptance tests are cool, these days!

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "query-params",
     "query params"
   ],
+  "repository": "https://github.com/ecraft/ember-query-params-proxy",
   "license": "MIT",
   "author": [
     "Mikael Riska <mikael.riska@ecraft.com>"
```
Comment (mriska) [inline package.json]: Feel free to add your name here since you are now also a maintainer :)

File: .eslintrc.js [hunk 2]
```diff
@@ -16,19 +16,28 @@ module.exports = {
   },
   rules: {
   },
+  globals: {
+    "visit": true,
+    "andThen": true,
+    "fillIn": true,
+    "currentURL": true
+  },
```
Comment (mriska) [inline .eslintrc.js]: Can we remove these now that you are importing them?

**Review by mriska — APPROVED** [review]

**Pull request merged**